pillar/backups/mitx.sls

```diff
@@ -57,6 +54,6 @@ backups:
         - curl
         - nfs-common
       settings:
-        duplicity_passphrase: {{ duplicity_passphrase }}
+        duplicity_passphrase: __vault__::secret-operations/global/duplicity-passphrase>data>value
```

Comment:
Since we're calling this multiple times, would it not be better if we set `duplicity_passphrase` at the beginning and then reference it throughout?